Makefile

```diff
@@ -336,6 +336,7 @@ TESTS = \
 	inlineskiplist_test \
 	env_basic_test \
 	env_test \
+	hash_test \
```

Comment:
I updated `CMakeLists.txt`. `.git_ignore` does not seem to need any update, the rule there is `*_test`.